src/packages/utils/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## v0.1.0 (2020-10-31)
+
+First release of `@excalidraw/utils` to provide utilities functions.
+
+- Added `exportToBlob` and `exportToSvg` to export an Excalidraw diagram definition, respectively,
+  to a [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob) and
+  to a [SVGElement](https://developer.mozilla.org/en-US/docs/Web/API/SVGElement (#2246)
```

Comment:
`#2246` can be removed as it's not adding the pr link and also would be better to have pr link at the top so it's easily discoverable.

Reply:
but a version can have multiple changes/pull requests?

Reply:
That actually depends on how we want to do the release, it can be one release per pr(if release needed) or multiple pr as you said so let's keep the pr link with changes in that case and we can move to the header if we go with former.
You can just make it a link and that should be fine.

Reply:
I think it's fine as is, and we should keep the date in the header for easier orientation. I'm doing the same in top-level changelog (minus the version), and (very) roughly try to follow https://keepachangelog.com/en/1.0.0/.

Reply:
Ok, let's keep the date then, my only worry was we might miss updating the date in case pr is not merged on the same day unless automated using something like [sematic-release](https://github.com/semantic-release/semantic-release/releases). But we should add pr links for every change so it's easier to find the changes.

Reply:
Technically, the date should be added just before the release. This change should be added in the "Unreleased" section:


```markdown
# Changelog

## [Unreleased]

- ...
```

Reply:
sounds good 👍
@Mogztter looks like push is disabled currently do you mind ticking the check box `Allow edits from maintainers` so we can push any changes before merging if needed :)?

Reply:
I don't know why but I don't have the checkbox... I cannot check "Allow edits from maintainers" :disappointed:

Reply:
That's weird 😕 👀 . It always shows up [here](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

Reply:
![view](https://user-images.githubusercontent.com/333276/97809577-ec83b680-1c6d-11eb-9233-5911ac18ab4b.png)